package/utils/helpers.js

```diff
@@ -33,16 +33,14 @@ const chdirTestApp = () => {
 
 const chdirCwd = () => process.chdir(process.cwd())
 
-const removeTrailingSlash = (path) => {
-  const hasTrailingSlash = path.endsWith('/')
-  if (hasTrailingSlash) return path.substr(0, path.length - 1)
-  return path
-}
+const resetEnv = () => (process.env = {})
```

Comment:
https://travis-ci.org/rails/webpacker/jobs/465694990#L703
Your test are pointing out that you are returning the value of the assignment. I would omit this altogether.